modules/picard/sortvcf/main.nf

```diff
@@ -0,0 +1,34 @@
+process PICARD_SORTVCF {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::picard=2.26.10" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/picard:2.26.10--hdfd78af_0' :
+        'quay.io/biocontainers/picard:2.26.10--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(vcf)
+
+    output:
+    tuple val(meta), path("*_sorted.vcf"), emit: vcf
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    picard \\
+        SortVcf \\
+        --INPUT $vcf \\
+        --OUTPUT ${prefix}_sorted.vcf
```

Comment:
is there an option for compressing the output vcf file?

Reply:
No, this tool doesn't have that option.

Reply:
You can use `gzip` in the container/env to compress the output.

Reply:
Fixed it 👍🏻 All I had to do was change the output file's suffix from .vcf to .vcf.gz 🤷🏻‍♂️ 🤦🏽